exporter/lambda/cross_region_import_replication.py

```diff
@@ -5,92 +5,73 @@
 
 import boto3
 import botocore
-from raven import Client
-from raven.transport import HTTPTransport
+import sentry_sdk
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+
+sentry_sdk.init(integrations=[AwsLambdaIntegration(timeout_warning=True)])
 
 MAX_RESOURCES_PER_TEMPLATE = 200
-RESSOURCE_BY_GROUP = 5
 
 
 def lambda_handler(*_):
     try:
         _lambda_handler()
     except:
-        # Using a the default transport does not work in a Lambda function.
-        # Must use the HTTPTransport.
-        Client(
-            dsn=os.environ['SENTRY_DSN'],
-            environment=os.environ['SENTRY_ENVIRONMENT'],
-            transport=HTTPTransport
-        ).captureException()
-        # Must raise, otherwise the Lambda will be marked as successful, and the exception
-        # will not be logged to CloudWatch logs.
         raise
 
 
 def _lambda_handler():
-    dynamodb_resource = boto3.resource('dynamodb')
-    cross_stack_ref_table = dynamodb_resource.Table(os.environ['CROSS_STACK_REF_TABLE_NAME'])
+    dynamodb_resource = boto3.resource("dynamodb")
+    cross_stack_ref_table = dynamodb_resource.Table(os.environ["CROSS_STACK_REF_TABLE_NAME"])
 
     scan_response = cross_stack_ref_table.scan()
-    cross_stack_references = scan_response['Items']
+    cross_stack_references = scan_response["Items"]
 
-    while scan_response.get('LastEvaluatedKey'):
-        scan_response = cross_stack_ref_table.scan(ExclusiveStartKey=scan_response['LastEvaluatedKey'])
-        cross_stack_references.extend(scan_response['Items'])
+    while scan_response.get("LastEvaluatedKey"):
+        scan_response = cross_stack_ref_table.scan(ExclusiveStartKey=scan_response["LastEvaluatedKey"])
+        cross_stack_references.extend(scan_response["Items"])
 
     if cross_stack_references:
-        number_of_chunk = len(cross_stack_references) / MAX_RESOURCES_PER_TEMPLATE
-        max_group_size = int(max(min(RESSOURCE_BY_GROUP/number_of_chunk, RESSOURCE_BY_GROUP), 1))
-
         nested_template_urls = []
         for items in _chunks(cross_stack_references, MAX_RESOURCES_PER_TEMPLATE):
-            nested_template_urls.append(_generate_nested_template(items, max_group_size))
+            nested_template_urls.append(_generate_nested_template(items))
```

Comment:
(one line above) rename `MAX_RESOURCES_PER_TEMPLATE` for `MAX_OUTPUTS_PER_TEMPLATE`